pkg/schema/load/dashboard.go

```
@@ -43,9 +51,13 @@ func BaseDashboardFamily(p BaseLoadPaths) (schema.VersionedCueSchema, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	currentDir, err := os.Getwd()
+	if err != nil {
+		fmt.Println(err)
```

Shouldn't we return an error here instead?